src/ontology/mondo-edit.obo

```diff
@@ -349180,18 +349180,9 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/6657/chronic-
 
 [Term]
 id: MONDO:0018335
-name: deep dermatophytosis
-comment: Reason: duplicate. This will be merged with MONDO:0021660
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:397587"}
-synonym: "disseminated granulomatous dermatophytosis" EXACT [Orphanet:397587]
-xref: ICD10:B35.8 {source="Orphanet:397587", source="ORDO:397587/ntbt"}
-xref: Orphanet:397587 {source="MONDO:equivalentTo"}
-xref: UMLS:C1395264 {source="MONDO:relatedTo", source="Orphanet:397587"}
-is_a: MONDO:0002041 {source="Orphanet:397587"} ! fungal infectious disease
-relationship: has_modifier MONDO:0021136 {source="MONDO:0015578"} ! rare
-relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4128
-property_value: IAO:0006012 "2022-03-01" xsd:string
+name: obsolete deep dermatophytosis
+is_obsolete: true
+replaced_by: MONDO:0021660
```

Comment:
could you please add the seeAlso and has obsolescence reason to the obsoleted term?
thank you! 
(if this is not in the documentation, could you add it there too? Thanks!)